.idea/uiDesigner.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should be deleted...it was accidentally checked in but does not belong to this project.

Reply:
removed and ignored